pom.xml

```diff
@@ -3982,4 +3982,5 @@
       <url>file:///tmp</url>
     </site>
   </distributionManagement>
+  <!-- tickle the build -->
```

Comment:
Remove this line?

Reply:
Yes, this line would not be merged. It's here to provoke a test run; without it, Yetus won't run over any maven modules.